tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -330,6 +330,13 @@ public:
    ENTupleStructure GetStructure() const { return fStructure; }
    std::size_t GetNRepetitions() const { return fNRepetitions; }
    NTupleSize_t GetNElements() const { return fPrincipalColumn->GetNElements(); }
+   /// Returns the number of elements that comprise a value for this field, which depends on the repetitions of each
+   /// parent field (see also `GetNRepetitions()`).  E.g., given the field hierarchy representing the type
+   /// `std::array<std::array<float, 4>, 2>`, this function shall return 8 for the inner-most field.
+   std::size_t GetNElementsPerEntry() const;
+   /// Returns the global index of the first entry that has a stored on-disk value. For late added fields, this allows
+   /// for differentiating zero-initialized values read before the addition of the field from actual stored data.
+   NTupleSize_t GetGlobalRangeFirst() const;
```

Comment:
I thought it would be useful from the users' perspective, taking into account that we don't (and probably won't) allow specifying any default value for entries before the field arrived.  This way, we give the user a chance to do it on his/her side.

Reply:
Per discussion on the EP R&D meeting on Wed 17th May, this function may be quite useful; it has been renamed to `GetFirstEntry()`.